bin/src/main.rs

```diff
@@ -390,32 +391,11 @@ async fn main() {
                             let RetryItem {
                                 body_buffer,
                                 offsets,
-                                path,
+                                path: _,
                             } = item;
                             match client.send(body_buffer, offsets).await {
                                 Ok(s) => match s {
-                                    SendStatus::Sent => {
```

Comment:
After refactoring and tracing down some failure on the disk used metric, I concluded that the file in question has already been deleted at this point.